docs/apps/qiime.md

```diff
@@ -37,46 +37,90 @@ module load qiime1
 ```
 
 To use QIIME2
-```text
-module load qiime2
+
+Check available versions with
+
+```
+module spider qiime2
 ```
 
-After that you can start Qiime2 with command:
+Load desired version with e.g.
+
+```
+module load qiime2/2023.9-amplicon
+```
+
+After that you can start QIIME2 with command:
 ```text
 qiime
 ```
 
-Please check Qiime2 home page for more instructions.
+## Distributions
 
-Note that many Qiime tasks involve heavy computing. Thus, these tasks should be executed as
-batch jobs. Qiime needs to have access to a local node specific file system for handling temporary data.
-This kind of directory is available on the NVMe nodes of Puhti. Therefore, you must include a request for NVMe space
-in your batch job file.
+Latest versions of QIIME2 come in different distributions: amplicon/shotgun/tiny.
+These distributions vary on which plugins come with them. You can compare the
+[distributions](https://docs.qiime2.org/2023.9/install/#distributions) on QIIME2
+home pages.
 
-The easiest way to start using Qiime is to use command `sinteractive` to launch an interactive batch job:
-```text
-sinteractive -i
+CSC provides installations for the amplicon and shotgn distributions.
+
+## Additional plugins
+
+CSC only maintains the basic distributions of QIIME2. If you need plugins not included in the basic distributions, you will need to install your own QIIME2.
+
+This can be easily done with the [Tykky tool](../computing/containers/tykky.md).
```

Comment:
This line perhaps can be merged with earlier paragraph as : "...you will need to install your own QIIME2 using [Tykky wrapper tool](../computing/containers/tykky.md).